File: src/main/java/ravex/modules/render/CustomFog.java
package ravex.modules.render;

import ravex.modules.Category;
import ravex.modules.Module;
import ravex.parameter.ColorParameter;

public class CustomFog extends Module {
    public static final CustomFog INSTANCE = new CustomFog();

    public final ColorParameter color = new ColorParameter("Fog Color", 0xFFFF5500);

    private CustomFog() {
        super("CustomFog", Category.RENDER);
        addParameter(color);
    }
}

File: src/main/kotlin/volthack/modules/render/CustomFog.kt
package volthack.modules.render

import volthack.setting.Category
import volthack.setting.Module

object CustomFog : Module("CustomFog", "Customizes fog and environment rendering", Category.RENDER) {
    val fogColor by color("Fog Color", 0xFF6C63FF.toInt(), "The color of the world fog")
}
